packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -70,7 +61,7 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   }, []);
 
   return (
-    <EditorContainer bg="black" as="main" isIFrame={isIFrame}>
+    <EditorContainer bg="black" height="25rem" width="43rem" {...rest}>
```

Comment:
I think it's better to put the  `minHeight='25rem'` and `maxWidth='43rem'` here so we can easily override them. Tried in storybook and it works nicely for a single page component with:

```
  <Story
    name="Portal"
    args={{
      snippetsBaseUrl: process.env.CONTAINER_API_BASE,
      maxWidth: '100%',
      width: '100%',
      height: '100vh'
    }}
  >
  ```

Reply:
wait you mean remove from editorBaseStyles and put in the styleguide story?

Reply:
Best option is to remove from `editorBaseStyles` and keep as props in the `EditorContainer`. 

Storybook is just for testing how consumers can use it! I was just testing it locally. :)

Reply:
we spoke offline and moved maxwidth to a component prop!